### PR TITLE
Feature/updated docker testing docs

### PR DESCRIPTION
**Pull request by schmunk42**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Related issues  | #15345

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: tests/.env-dist [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #COMPOSE_FILE=docker-compose.yml:docker-compose.caching.yml
 #COMPOSE_FILE=docker-compose.yml:docker-compose.mysql.yml
 #COMPOSE_FILE=docker-compose.yml:docker-compose.pgsql.yml
+#COMPOSE_FILE=docker-compose.yml:docker-compose.mysql.yml:docker-compose.pgsql.yml:docker-compose.caching.yml
```
Comment (schmunk42) [inline tests/.env-dist]: Should we uncomment this line by default, so if a developer copies this file to `.env` he get's a full featured test environment?

Comment (samdark) [reply tests/.env-dist]: Yes, I think so.

**Comment by schmunk42** [comment]
PR Updated.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**